X16 Reference - 12 - Hardware.md

```diff
@@ -0,0 +1,133 @@
+## Chapter 12: Hardware Pinouts
+
+This chapter covers pinout for the I/O ports and performing firmware updates on the SMC, VERA, and system ROMs.
+
+### Port and Socket Listing
+* VERA Connectors
+  * VERA Mainboard socket
+  * SD Card
+  * Composite Video
+  * S-Video
+  * VGA
+* SNES Controller Ports (x2)
+* IEC Port
+* PS/2 Keyboard and mouse
+* Expansion Ports (x4)
+* User Port header
+* ATX Power Supply
+* Front Panel
+
+## Disclaimer
+
+The instructions and information in this document are the best available information at the time of writing. This information is subject to change, and no warranty is implied. We are not liable for damage or injury caused by use or misuse of this information, including damage caused by inaccurate information. Interfacing and modifying your Commander X16 is done solely at your own risk.
+
+If you attempt to upgrade your firmware and the process fails, one of our community members may be able to help. Please visit the forums or the Discord community, both of which can be reached through https://commanderx16.com.
+
+### VERA Connectors
+
+TODO
+
+### SNES Ports
+
+The computer contains two SNES style ports and will work with Super Nintendo compatible game pads. 
+
+<img src="images/SNES_Controller_Female.svg" alt-text="SNES Controller Port" width="256" />
```

Comment:
If `width` isn't important, could this be re-written directly in markdown?
```
![SNES Controller Port](images/SNES_Controller_Female.svg)
```

Reply:
It could be, but I’d have to re-scale the images. I will probably do that anyway, but I don’t have very good SVG fools right now.

Reply:
Sounds good.  I checked around to see how other images were placed and the only one I found was the VERA palette.  Hard to say that constitutes a standard.  Tags are probably fine unless someone else has a concern.